kotsadm/web/src/components/apps/AppLicense.jsx

```diff
@@ -122,12 +120,6 @@ class AppLicense extends Component {
       })
       .catch(err => {
         console.log(err);
-        err.graphQLErrors.map(({ msg }) => {
-          this.setState({
-            message: msg,
```

Comment:
should we keep this with something like `err ? error.message : "something went wront"` ?